app/views/_components/offer-panel/template.njk

```diff
@@ -5,7 +5,7 @@
   {% if params.withdraw or params.defer %}
     <p class="govuk-body">
       {% if params.withdraw %}
-        {# <a href="{{params.withdraw.href}}">Withdraw offer</a> #}
+        <a href="{{params.withdraw.href}}">Withdraw offer</a>
```

Comment:
Lets leave this for a separate commit.

Reply:
I've commented it out from the offer page, so this won't be seen.